H5P.BranchedVideo/branched-video.js

```diff
@@ -57,12 +60,32 @@ H5P.BranchedVideo = (function ($) {
         videoDiv.style.position = 'relative';
         var video = document.createElement('video');
         video.id = 'tapestry-video-' + this.slug;
-        video.src = this.source;
+        //video.src = this.source;
```

Comment:
Do you need to remove this instead? If you are going to keep some code commented, please add a comment before to explain why this is commented out.